modules/oc-adm-by-example-content.adoc

```diff
@@ -395,11 +419,11 @@ Drain node in preparation for maintenance
 .Example usage
 [source,bash,options="nowrap"]
 ----
-  # Drain node "foo", even if there are pods not managed by a ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet on it.
-  $ oc adm drain foo --force
+  # Drain node "foo", even if there are pods not managed by a replication controller, replica set, job, daemon set or stateful set on it
```

Comment:
This list needs an Oxford comma after "daemon set" and before "or" per the IBM style guide.